ecommers/lib/core/common/validator/shipping_address_validator.dart

```diff
@@ -0,0 +1,20 @@
+class ShippingValidator{
+
+  static bool isValidFullName(String fullName) =>
+      fullName != null && fullName.length > 3;
+
+  static bool isValidAddress(String address) =>
+      address != null && address.length > 2;
+
+  static bool isValidCity(String city) =>
+      city != null && city.length > 2;
+
+  static bool isValidState(String state) =>
+      state != null && state.length > 3;
```

Comment:
can you cheek to work state?.lenght>3;

Reply:
the ' > ' method cannot be called on null